examples/library/library_example_test.go

```diff
@@ -36,11 +36,8 @@ g.from(start: 1993-02-16T00:00:00Z, stop: 1993-02-16T00:03:00Z, count: 5, fn: (n
 		panic(err)
 	}
 
+	ctx = executetest.NewTestExecuteDependencies().Inject(ctx)
```

Comment:
Should this be changed to `InjectTestExecuteDependencies`?

This is a fairly large change and I'd like to reduce moving parts so please don't change this at the moment, but we can consider this in a follow-up PR.

Reply:
Yep, could be an improvement.